src/main.tsx

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/default
+import React from 'react'
```

Comment:
This rule is stupid because it applies to stuff that does not depend on us. I think we should remove it (and make sure it applies on *exports* instead).
Not this PR's business though 🤷

Reply:
Agreed. But out of scope, let's deal with this later.